integration/shareddata/scalars.go

```diff
@@ -134,20 +138,22 @@ var Doubles = &Values[string]{
 		"double-big-plus":  doubleBig + 1,
 		"double-big-minus": doubleBig - 1,
 
-		// double max precision ~1<<53
-		"double-prec-max":       doubleMaxPrec,
-		"double-prec-max-plus":  doubleMaxPrec + 1,
-		"double-prec-max-minus": doubleMaxPrec - 1,
+		// double max precision ~1<<53 - 1
+		"double-prec-max":          doubleMaxPrec,
+		"double-prec-max-plus":     doubleMaxPrec + 1,
+		"double-prec-max-plus-two": doubleMaxPrec + 2,
+		"double-prec-max-minus":    doubleMaxPrec - 1,
 
 		// negative double big values ~ -(1<<61)
 		"double-neg-big":       -doubleBig,
-		"double-neg-big-plus":  -(doubleBig + 1),
-		"double-neg-big-minus": -(doubleBig - 1),
+		"double-neg-big-plus":  -doubleBig + 1,
+		"double-neg-big-minus": -doubleBig - 1,
 
 		// double min precision ~ -(1<<53 - 1)
-		"double-prec-min":       -(doubleMaxPrec - 1),
-		"double-prec-min-plus":  -(doubleMaxPrec - 1) + 1,
-		"double-prec-min-minus": -(doubleMaxPrec - 1) - 1,
+		"double-prec-min":           -doubleMaxPrec,
+		"double-prec-min-plus":      -doubleMaxPrec + 1,
+		"double-prec-min-minus":     -doubleMaxPrec - 1,
+		"double-prec-min-minus-two": -doubleMaxPrec - 2,
```

Comment:
That's the one we didn't have before. I'm ok to have both `minus one` and `minus two`, but then let's also have `plus one` and `plus two` as `plus two` is something we have in the other places of this PR.

And if we have `minus two` here (if there is a reason for it), should we also have it in the other places?

Reply:
I don't think that `+2` is necessary for `double-prec-min` cases. The point of doing a check for `-2` values is related to the fact that because of lost precision `double-prec-min`-1 == `double-prec-min`-2. The same applies for `double-prec-max` **+**`2`

Reply:
yeah, that sounds reasonable!

Reply:
Ah I see, got it too